### PR TITLE
Add new option in espanol/huida/pagar

### PR DESCRIPTION
**Pull request by TXinTXe**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2288)

### CODE REVIEW

**Comment by TXinTXe** [comment]
Please review this, @casey-collab

File: espanol/huida/pagar/pagar.md [hunk 1]
```diff
@@ -1 +1,5 @@
 Tras aceptar tus dos monedas de oro, el hombre te guia a través de un túnel hasta las afueras de la ciudad.
+Despues te deja salir y cierra la puerta tras de si.
+Estas solo. Ves un camino que se pierde en la distancia hacia el oeste.
+[Decides tomar el camino.](/camino/camino.md)
```
Comment (casey-collab) [inline espanol/huida/pagar/pagar.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: espanol/huida/pagar/camino/camino.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Andas por el camino durante horas pero parece no acabar nunca.
```
Comment (casey-collab) [inline espanol/huida/pagar/camino/camino.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/huida/pagar/pagar.md (hunk 1 above)
Comment (TXinTXe) [inline espanol/huida/pagar/pagar.md]: Please review this, @casey-collab

File: espanol/huida/pagar/camino/camino.md (hunk 2 above)
Comment (TXinTXe) [inline espanol/huida/pagar/camino/camino.md]: Please review this, @casey-collab

**Comment by TXinTXe** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :wind_chime:

**Pull request merged**